tests/std/test.lst

```diff
@@ -212,7 +212,7 @@ tests\P0433R2_deduction_guides
 tests\P0487R1_fixing_operator_shl_basic_istream_char_pointer
 tests\P0513R0_poisoning_the_hash
 tests\P0553R4_bit_rotating_and_counting_functions
-tests\P0556R3_bit_ispow2_ceil2_floor2_log2p1
+tests\P0556R3_integral_power_of_two_operations

```

Comment:
Nitpick: Should this say `_bit_` at the beginning?

Reply:
yeha, probably. applied